playbooks/common/openshift-cluster/upgrades/v3_9/upgrade.yml

```diff
@@ -22,6 +22,12 @@
 
 - import_playbook: validator.yml
 
+# Install the web console before upgrading the masters since 3.9 masters will
+# immediately begin proxying to the webconsole service.
+- import_playbook: ../../../../openshift-web-console/private/config.yml
```

Comment:
I'm not sure this is the right spot for this playbook to be included.

Reply:
Where should I include it?

Reply:
Lets move it https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/upgrades/post_control_plane.yml this is the phase after the control plane has been updated and policy has been reconciled.